src/app/src/util/constants.oarmap.js

```diff
@@ -2,7 +2,7 @@ import env from './env';
 
 export const MAPBOX_TOKEN = env('REACT_APP_MAPBOX_TOKEN');
 
-export const CIRCLE_COLOR = '#3d2f8c';
+export const CIRCLE_COLOR = '#0427a4';
```

Comment:
This file has been deleted in the switch to Google Maps PR. https://github.com/open-apparel-registry/open-apparel-registry/pull/284 If that is merged first you may run into a conflict. 

Once the Google Maps PR is merged we will need some follow up design work to make markers and clusters that match the branding.

Reply:
We should move these values into the `COLOURS.js` file and get them from there.

Reply:
I moved these during a rebase.